### PR TITLE
make BackButton more accessable

### PR DESCRIPTION
**Pull request by Sala7din**
remove dublicate classes and turn BackButton from link to real Button

### CODE REVIEW

**Review by pascalwengerter — APPROVED** [review]
Thanks, good find!

File: src/components/BackButton.vue [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <template>
```
Comment (pascalwengerter) [inline src/components/BackButton.vue:1]: Please note that my approval for this PR is limited to the changes to the back button (https://github.com/pascalwengerter/musclemate-frontend/pull/32/commits lists three individual commit, I'd ask you to remove the other two commits from this branch ;) )

File: src/store/store.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ export const store = createStore({
     name: "Vue",
     workout: {
       duration: {
-        active: "30",
+        active: "60",
```
Comment (pascalwengerter) [inline src/store/store.js:8]: This is only the default value and I'd prefer it to stay at 30s - feel free to open an issue asking about a "default duration defined within each workout" so we can lift it to 60s for your ones :)

File: src/workouts.json [hunk 3]
```diff
@@ -1,86 +1,115 @@
 {
-    "Classic Workout": [
+    "BJJ Mobility Warm Up Routine": [
```
Comment (pascalwengerter) [inline src/workouts.json:2]: Currently, you're replacing the two existing workouts with the BJJ+Yoga routines - this change I can't accept in my `main` branch, I'd be happy to approve them if you'd just add them to the bottom if this list (so we'd have 4 workouts in total to choose from) 😎

File: docs/notes/workouts.json.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+---
+id: dpeysudtg6of05oz7odgd4a
+title: Json
+desc: ''
+updated: 1678569579309
+created: 1678569523359
+---
+**Questions**
+
+1. What does the boolean in the Array items do and why are some items set to false and others to true?
```
Comment (pascalwengerter) [inline docs/notes/workouts.json.md:10]: This makes for a great issue where we can discuss where this should be heading in the future - apart from it, I think the whole `docs`/`dendron` changes were accidentally commited to the branch?

**Review by pascalwengerter — CHANGES_REQUESTED** [review]
I've added some comments to this PR, let me know if questions arise regarding them

File: src/components/BackButton.vue (hunk 1 above)
Comment (Sala7din) [reply src/components/BackButton.vue:1]: Got it, I'll remove the two commits that are not related to the changes made to the back button, as per your request.

File: src/store/store.js (hunk 2 above)
Comment (Sala7din) [reply src/store/store.js:8]: I appreciate that the default value is preferred to stay at 30s. I will open an issue to request an increase to 60s for my specific workouts.

File: docs/notes/workouts.json.md (hunk 4 above)
Comment (Sala7din) [reply docs/notes/workouts.json.md:10]: Thanks for the feedback. I agree to add the new routines to the bottom of the list. I'll make the changes and submit them for your review. Thanks again for your help

**Comment by pascalwengerter** [comment]
Closing as superseded by #37, so the relevant commit will make it to the `main` branch without you having to invest more time/energy here 🥳

**Pull request closed without merging**